appeals/api/src/server/openapi-types.ts

```diff
@@ -1875,3 +1875,8 @@ export interface UpdateServiceUserResponse {
 	/** @example 1 */
 	serviceUserId?: number;
 }
+
+export interface WithdrawalRequestRequest {
```

Comment:
Maybe just `WithdrawalRequest`?

Reply:
`WithdrawalRequest` is the name of the object, the appending `Request` is an indicator it is a request and not a response

Reply:
Perhaps technically that's right, but surely this can be avoided? It can be seen as a mistake fairly easily.

Reply:
I think it's minor